docs/frontend/keras.rst

```diff
@@ -1,11 +1,19 @@
 ================
-Keras and QKeras
+Keras and its quantized variants
 ================
 
-Keras and the quantization library QKeras are well supported in ``hls4ml``. Currently, the Keras v2 (``tf.keras``) is the preferred version, and the future versions of ``hls4ml`` will expand support for Keras v3. The frontend is based on the parsing the serialized json representation of the model.
+Keras and the quantization library QKeras are well supported in ``hls4ml``. Both Keras v2 (``tf.keras``) and the new Keras v3 are supported. While the Keras v2 support is based on parsing the serialized json representation of the model, the Keras v3 support uses direct model inspection.
```

Comment:
I am not sure if the claim that the v2 support is based on parsing the serialized json representation is always true.

Reply:
It is. If a model is a live object we will parse the `Model.to_json()`. If the model is an .h5 file we will extract the `model_arch.json` from the archive and parse that.